src/Raven.Server/Documents/Replication/ReplicationLoader.cs

```diff
@@ -970,7 +970,7 @@ private void HandleMigrationReplication(DatabaseRecord newRecord, List<IDisposab
                 if (handler is not OutgoingMigrationReplicationHandler migrationHandler)
                     continue;
 
-                if (newRecord.ShardBucketMigrations.TryGetValue(migrationHandler.BucketMigrationNode.Bucket, out var migration) == false)
+                if (newRecord.Sharding.ShardBucketMigrations.TryGetValue(migrationHandler.BucketMigrationNode.Bucket, out var migration) == false)
```

Comment:
nullable?

Reply:
Shouldn't be possible, we are already in shared db here, but added to be safe.